src/Components/Web.JS/src/Rendering/BrowserRenderer.ts

```diff
@@ -370,23 +382,28 @@ export class BrowserRenderer {
     }
 
     switch (element.tagName) {
-      case 'INPUT':
-      case 'SELECT':
-      case 'TEXTAREA': {
+      case 'INPUT': {
         const value = attributeFrame ? frameReader.attributeValue(attributeFrame) : null;
+        element[inputValuePropname] = value;
```

Comment:
Could you have the comment from earlier over here (so that it's consistent with with `select`)?